features.json

```diff
@@ -78,13 +78,14 @@
 			"logo": "/images/firefox.svg",
 			"version": "77",
 			"features": {
-				"bigInt": "Nightly",
-				"bulkMemory": "javascript.options.shared_memory",
-				"multiValue": "Nightly",
+				"bigInt": "Beta 78",
```

Comment:
This is enabled by default since Firefox 78, so Nightly is not correct status

Reply:
Is it now available via flags on stable as well perhaps?

Reply:
I can't find any related wasm flags on stable

<img width="1307" alt="image" src="https://user-images.githubusercontent.com/1507086/85613687-ce280180-b662-11ea-92a5-5ed82acac835.png">

Screen from Firefox 78 with all flags with default values

<img width="697" alt="image" src="https://user-images.githubusercontent.com/1507086/85613909-0b8c8f00-b663-11ea-8b48-8cd2340428cd.png">

Reply:
comment about BigInt support in Beta https://bugzilla.mozilla.org/show_bug.cgi?id=1631702#c1
patch for multi-value support in Beta https://bugzilla.mozilla.org/show_bug.cgi?id=1628321#c3

Reply:
Sounds good, thanks for checking!